diff --git config/afp.conf.tmpl config/afp.conf.tmpl
index b2ee186..754d99e 100644
--- config/afp.conf.tmpl
+++ config/afp.conf.tmpl
@@ -5,8 +5,25 @@
 [Global]
 ; Global server settings
 
+; enable spotlight and correct the dbus daemon path
+dbus daemon = /usr/bin/dbus-daemon
+spotlight = yes
+
+; provide AFP runtime statistics (connected users, open volumes) via dbus.
+afpstats = yes
+
+; enable guest access as well as user accounts
+uam list = uams_guest.so uams_dhx2.so uams_dhx.so
+
+; output log entries to stdout instead of syslog
+log file = /dev/stdout
+log level = default:note
+
 ; [Homes]
 ; basedir regex = /xxxx
 
-; [My AFP Volume]
-; path = /path/to/volume
+[Repos]
+path = /repos
+
+[Builds]
+path = /builds
\ No newline at end of file
